### DISCUSSION TITLE
Fake GC-01: Support

### DISCUSSION THREAD

**Discussion by RemoteMedic** [Gissio/radpro#58] [Q&A] [answered]
So, I've been trying to update the firmware on my 'FNIRSI' GC-01 with no luck. Tried many cables and operating systems, USB2.0 etc. I took a look at the board that's in my unit and it looks like nothing seen in this  group. I think it's a clone or fake unit. It has the correct markings on the back and front of the enclosure but the inside is completely different, even down to the battery.

Could you confirm if this unit is legit please? If not, I'll return it to AliExpress and buy one from Amazon.

Many thanks in advance, RemoteMedic - Chris UK
![IMG_9642](https://github.com/Gissio/radpro/assets/166464405/b76fb714-4818-4752-b976-b66b40ba6136)

**Comment by Gissio** [comment, marked as answer]
Hello there!

The device seems to be the same as this one: https://github.com/Gissio/radpro/issues/48

In that thread, a user reported finding many signs of a fake.

**Comment by J0s3f** [comment]
I got the same device from AliExpress. The seller denied it's a fake and said it's just an "older version".
After contacting FNIRSI using the email address on their official website, they confirmed that they did not produce this device, and I got the money back from AliExpress.
